lib/origin_simulator.ex

```diff
@@ -19,6 +20,20 @@ defmodule OriginSimulator do
     |> send_resp(200, Poison.encode!(msg))
   end
 
+  get "/clear_current_count" do
```

Comment:
probably a `post` would be more appropriate but for sure less convenient

Reply:
that was my thinking too...

Reply:
convenience 🆚 conventions :)